pkg/kube/controllers/extendedstatefulset/volumes.go

```diff
@@ -154,11 +157,36 @@ func (r *ReconcileExtendedStatefulSet) generateVolumeManagementSingleStatefulSet
 	statefulSet.SetLabels(labels)
 	statefulSet.SetAnnotations(annotations)
 
+	statefulSet.Spec.PodManagementPolicy = v1beta2.ParallelPodManagement
+
+	// Generate dummy volumeMounts
+	// "volumeClaimTemplates: [...] Every claim in this list must have at least one matching (by name) volumeMount in one container in the template."
+	// See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#statefulsetspec-v1-apps
+	volumeMounts := make([]corev1.VolumeMount, 0, len(statefulSet.Spec.VolumeClaimTemplates))
+	for _, pvc := range statefulSet.Spec.VolumeClaimTemplates {
+		volumeMounts = append(volumeMounts, corev1.VolumeMount{
+			Name:      pvc.Name,
+			MountPath: path.Join("/mnt", pvc.Name),
+		})
+	}
+
+	statefulSet.Spec.Template.Spec.InitContainers = []corev1.Container{}
+	statefulSet.Spec.Template.Spec.Containers = []corev1.Container{{
+		Name:            "volume-management",
+		VolumeMounts:    volumeMounts,
+		Image:           converter.GetOperatorDockerImage(),
+		ImagePullPolicy: converter.GetOperatorImagePullPolicy(),
+		Args: []string{
+			"util",
+			"sleep",
```

Comment:
Is it  really necessary to add `sleep` as a command? Wouldn't it be sufficient to shell out and call `bash -c 'sleep N'` or even `read`? It's already a shell-out, so why not reuse something we know we have on our image?

Reply:
We thought this would be the pattern to use, but we can also call `bash` or `ruby` if you prefer.
Do you want us to change it?

Reply:
Yes, I would prefer it. We do control the whole docker image anyways. And if running standalone, bash should be available. Running the operator on Windows is not a thing.

Reply:
I used `ruby` which we need anyway for running the bosh templating logic (I guess).